cryoet_data_portal_neuroglancer/models/shader_builder.py

```diff
@@ -0,0 +1,110 @@
+"""Create GLSL shaders for Neuroglancer."""
+
+from typing import Any, Iterable, Optional
+
+TAB = "  "
+
+
+class ShaderBuilder:
```

Comment:
Note: the overall shader builder setup will be removed from this PR when the other PR with this builder is merged to main